solutionbox/inception/datalab_solutions/inception/_util.py

```diff
@@ -54,9 +55,14 @@ def get_train_eval_files(input_dir):
   return train_files, eval_files
 
 
-def get_labels(input_labels_file):
-  """Get a list of labels from labels file."""
-  with ml.util._file.open_local_or_gcs(input_labels_file, mode='r') as f:
+def get_labels(input_dir):
+  """Get a list of labels from preprocessed output dir."""
+  latest_file = os.path.join(input_dir, 'latest')
```

Comment:
consider moving the "get_latest_root_dir" to its own function so get_labels and get_train_eval_files can reuse the same code. Then you can also do error checking (what if the latest file does not exist, or if the folders referenced in it no longer exist?)

Your call.